modules/oe_content_event/tests/src/Kernel/EventLinkSourceFilterTest.php

```diff
@@ -164,6 +176,29 @@ public function testEventLinkSourceFilter(): void {
     ];
     $this->assertEqual($cache->getCacheTags(), $date_cache_tags);
 
+    // Set the time to test ongoing events.
```

Comment:
Looks like we are missing tests of edge cases when the current time is '2016-05-10T12:00:00' or '2050-05-15T12:00:00'. As I understand we are expecting at this moment anyway to have the ongoing event.

Reply:
Done.